src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/equalshashcode/InputEqualsHashCode.java

```diff
@@ -6,40 +6,40 @@
 
 package com.puppycrawl.tools.checkstyle.checks.coding.equalshashcode;
 
-public class InputEqualsHashCode { // ok
-    public boolean notEquals() { // ok
+public class InputEqualsHashCode { 
+    public boolean notEquals() { 
```

Comment:
```suggestion
    public boolean notEquals() {
```
There shouldn't be any trailing white-space at the end of line after removing the comment. Apply the changes for all such occurrence.